respa_admin/static_src/styles/form/navbar-form.scss

```diff
@@ -38,7 +38,7 @@
       }
 
       .language-item-selected {
-        border: solid 1px black;
+        border: solid 1px black !important;
```

Comment:
`!important` in CSS is usually bad practice.
It can be removed by changing this selector to `.language-item.language-item-selected` so it becomes more specific and the rule has higher priority.

Also change `border: ...` to `border-color: black`. That way other border settings are not overwritten here as we only need to change the color.

Reply:
Good point. Changed the priority by changing the order of the rules.